Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -0,0 +1,263 @@
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Helios2Sentinel;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+using System.Collections.Generic;
+using System.Collections;
+using System.Dynamic;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Text.Json;
+using System.Text;
+using System.Threading.Tasks;
+using System;
+
+namespace Helios2Sentinel
+{
+    public class IncidentProducer
+    {
+        private const string keyVaultName = "Cohesity-Vault";
+        private static readonly object queueLock = new object();
+        private static Lazy<ConnectionMultiplexer> lazyConnection = CreateConnection();
+        private static string containerName = GetSecret("containerName");
```

Comment:
You need to create a shared constant for the consumer and producer as we can't change the container name because playbooks refer to a pre-defined name "cohesity-extra-parameters"